src/lib/pubkey/curve25519/curve25519.cpp

```diff
@@ -88,7 +88,7 @@ Curve25519_PrivateKey::Curve25519_PrivateKey(const AlgorithmIdentifier&,
    size_check(m_private.size(), "private key");
    }
 
-secure_vector<byte> Curve25519_PrivateKey::pkcs8_private_key() const
+secure_vector<byte> Curve25519_PrivateKey::private_key_bits() const
    {
    return DER_Encoder()
       .start_cons(SEQUENCE)
```

Comment:
Seems ok when looking into https://tools.ietf.org/html/rfc5915 and https://tools.ietf.org/html/draft-ietf-curdle-pkix-00#page-6. `publicKey` field is optional, but a SHOULD.